tests/backpressure_test/Makefile

```diff
@@ -6,10 +6,13 @@ else # USING_XWAF
 # compiled for. It either refers to an XN file in the source directories
 # or a valid argument for the --target option when compiling
 
-ifeq ($(CONFIG),XCORE_AI)
+TARGET ?= XCORE-200-EXPLORER
+
+ifeq ($(XCORE_AI), 1)
+TARGET = XCORE-AI-EXPLORER
+endif
+ifeq ($(CONFIG), XCORE_AI)
```

Comment:
A question out of ignorance, what sets the `CONFIG` variable?  I can't see it in either the wscript or the Jenkinsfile.

Reply:
I don't know either. Looking at the source for xcommon it's also mentioned there, so I'm inclined to answer "xcommon magic" but I didn't want to touch it in case it broke something else, it being a system I don't yet fully understand.

Reply:
Okay, we will leave it in.

Reply:
The command line sets it. But yes, xcommon "magic".